code/_global_vars/lists/sound.dm

```diff
@@ -160,16 +160,21 @@ GLOBAL_LIST_INIT(sfx_list, list(
 		'sounds/items/Welder2.ogg',
 		'sounds/machines/airlock.ogg',
 		'sounds/effects/clownstep1.ogg',
-		'sounds/effects/clownstep2.ogg'
-		),
+		'sounds/effects/clownstep2.ogg',
+	),
 
 	SFX_SCP1507_ATTACK = list(
 		'sounds/scp/1507/attack1.ogg',
 		'sounds/scp/1507/attack2.ogg',
 		'sounds/scp/1507/attack3.ogg',
 		'sounds/scp/1507/attack4.ogg',
 		'sounds/scp/1507/attack5.ogg',
-		'sounds/scp/1507/attack6.ogg'
-	)
-
+		'sounds/scp/1507/attack6.ogg',
+	),
+
+	SFX_RATTLE = list(
+		'sounds/items/rattle1.ogg',
+		'sounds/items/rattle2.ogg',
+		'sounds/items/rattle3.ogg',
+	),
```

Comment:
Why did you just randomly decide to switch 1 file with lists to /tg/ list shit standard??????? but not others??? IN A PIGGYBANK PR?

Reply:
i saw an opportunity for code cleanup and i took it. although, yeah, i should probably make a separate PR doing this globally